### PR TITLE
KIALI-3132 Add documentation link in top right "?"-menu

### PR DESCRIPTION
**Pull request by josejulio**
** Describe the change **

- The links behaves different in upstream/downstream
       * Upstream, link has utm_* values appened to know which kiali
       version is requesting the documentation
       * Downstream doesn't have this utm_* values added

e.g. downstream links looks: https://www.kiali.io/documentation/ while upstreams: https://www.kiali.io/documentation/?utm_source=kiali&utm_medium=app&utm_campaign=v1.2.0-SNAPSHOT&utm_content=%3F-menu

** Issue reference **

https://issues.jboss.org/browse/KIALI-3132

** Backwards compatible? **

[ ] Is your pull-request introducing changes in behaviour?

Adds "Documentation" link to help(?) menu.

** Screenshot **

help-menu

### CODE REVIEW

**Comment by pilhuhn** [comment]
Can we make documentation the 1st option in the menu?

File: src/config/Config.ts [hunk 1]
```diff
@@ -39,11 +38,15 @@ const conf = {
...
     website: {
-      url: 'http://kiali.io',
+      url: 'https://www.kiali.io', // Without www, we get an SSL error
       icon: 'HomeIcon',
       linkText: 'Visit our web page'
     }
   },
+  /** */
+  documentation: {
+    url: 'https://www.kiali.io/documentation/' // Without www, we get an SSL error
```
Comment (pilhuhn) [inline src/config/Config.ts]: Can this be overwritten for downstream?

Comment (josejulio) [reply src/config/Config.ts]: I think we can take this value from the env file

**Comment by josejulio** [comment]
> Can we make documentation the 1st option in the menu?

Sure!

**Comment by josejulio** [comment]
Done, the menu is at the top and the url is configurable for upstream/downstream

Screenshot_2019-07-19_10-30-00

File: src/components/UpstreamDetector/UpstreamDetector.tsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
+
+interface UpstreamDetectorProps {
+  readonly upstream: React.Component;
+  readonly downstream: React.Component;
+}
+
+export const isUpstream = process.env.REACT_APP_IS_UPSTREAM === 'true';
+
+export default class UpstreamDetector extends React.PureComponent<UpstreamDetectorProps> {
```
Comment (israel-hdez) [inline src/components/UpstreamDetector/UpstreamDetector.tsx]: It's not being used at the moment, right?

Comment (josejulio) [reply src/components/UpstreamDetector/UpstreamDetector.tsx]: Right, in the end I didn't use this. Could be useful for https://issues.jboss.org/browse/KIALI-3131 or we could remove it.

**Pull request merged**